src/BuildScriptGenerator/DotNetCore/DotnetCorePlatform.cs

```diff
@@ -121,6 +121,9 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             manifestFileProperties[ManifestFilePropertyKeys.DotNetCoreSdkVersion]
                 = dotNetCorePlatformDetectorResult.SdkVersion;
 
+            string outputType = dotNetCorePlatformDetectorResult.OutputType;
+            AddOutputTypeOptionalField(manifestFileProperties, outputType);
```

Comment:
nit: you probably do not need a method to write outputtype to manifestfile unless you are writing an unit test to check how you are handling if outputtype is null/empty and so on